src/components/ToDoItem/Forms/FormForItem.js

```diff
@@ -2,23 +2,49 @@ import React from 'react'
 import '../ToDoItem.css'
 
 class FormForItem extends React.Component {
+	state = {
+		showButtons: false
+	}
+
+	handleHoverOn = () => {
```

Comment:
Nice logic.
Please notice that you can do it with css:
```
.edit-delete{ display: none; }
.todo-item:hover .edit-delete{ display: block; }
```

Reply:
Changed.